lust/android/key.properties

```diff
@@ -0,0 +1,4 @@
+storePassword=android
+keyPassword=android
+keyAlias=key
+storeFile=C:/Users/mateo/key.jks
```

Comment:
For Travis to build correctly I guess you should use a relative path.

Reply:
thanks a lot, I'll take a look to everything later

Reply:
btw before starting to work, at the moment the key is storaged locally in my computer;

 I guess the tutorial you sent me is to solve this problem, and make the key available for everyone in the repo right?

Reply:
That should work, yes. I just thought about to give Travis this possibility, but normally all the others should be able too.
Otherwise you can just send them the file and then use a relative path. But I think if it is working with the tutorial it is a good thing!